packages/saasify-cli/lib/cli.js

```diff
@@ -23,6 +23,8 @@ module.exports = async (argv, opts = { }) => {
     .option('-c, --config <path>', 'Path to `saasify.json` file (defaults to cwd)')
     .option('-C, --no-clipboard', 'Do not attempt to copy URL to clipboard')
 
+  // TODO: add ability to set client.teamId in global program options
```

Comment:
Captured?

Reply:
see my other comment about TODOs. agreed we should capture them, but there are so many and most of them i'd deem rally low priority like this one.